api/src/dto/_baseDto.ts

```diff
@@ -17,7 +17,7 @@ export class _baseDto {
     @IsEnum(DocType)
     type: DocType;
 
-    @IsDate()
+    // TODO check if it's a valid timestamp
```

Comment:
Set to @IsNumber for now (unless there is a better way to check if it is a Unix timestamp). Could perhaps check if the number is bigger than 1 January 2020 or so?